src/asap.ts

```diff
@@ -1,18 +1,17 @@
 import got from 'got';
+import express from 'express';
 import sha256 from 'sha256';
 import NodeCache from 'node-cache';
 import { Request } from 'express';
 import { secretType } from 'express-jwt';
 import { Logger } from 'winston';
 
-class ASAPPubKeyFetcher {
+export class ASAPPubKeyFetcher {
     private baseUrl: string;
     private ttl: number;
     private cache: NodeCache;
-    private logger: Logger;
 
     constructor(logger: Logger, baseUrl: string, ttl: number) {
```

Comment:
Still takes a logger in the constructor?

Reply:
Thanks, fixed.